polls/tests.py

```diff
@@ -1,10 +1,10 @@
 from datetime import date, datetime, timezone
 
-from backend.tests_utils import BaseTestCase
+from backend.tests_utils import WeakAuthenticationBaseTestCase
 from polls.models import Poll
 
 
-class PollTestCase(BaseTestCase):
+class PollTestCase(WeakAuthenticationBaseTestCase):
```

Comment:
Is there a way to avoid to change all `BaseTestCase` to `WeakAuthenticationBaseTestCase` ?

Reply:
I could have left it as `BaseTestCase`, but I thought changing the name was important as we are disabling a major security feature here, and we don't want to use `WeakAuthenticationBaseTestCase` for testing, well, the authentication security

Reply:
Is there a way not to remove theses security feature ?

Reply:
Another approach would be to replace the PUBLIC_KEY settings indeed
I'll do that